bsp/avr32/at32uc3a0256/board.c

```diff
@@ -78,6 +80,10 @@ void rt_hw_board_init(void)
     rt_hw_gpio_init();
 #endif
 
+#ifdef RT_USING_I2C
```

Comment:
this is bsp level, so it should use BSP_USINGxxx

```suggestion
#ifdef BSP_USING_SOFT_I2C
```

Reply:
Dear @mysterywolf,

Thank you very much. I am so sorry. I overlooked the config of the
channels. I have fixed it. Second: I also understand the configuration
hierarchy. It makes sense (BSP_* against RT_*). I will keep this in
mind for all the other hacks. I hope this fix is alright. Many thanks
again. Please let me know if I can help with anything else. Regards.